frontend/__tests__/components/CardScreen.test.tsx

```diff
@@ -2,52 +2,54 @@ import { act, render, screen } from '@testing-library/react'
 import CardScreen from '@/components/CardScreen'
 import userEvent from '@testing-library/user-event'
 
+const restaurants = [
+  {
+    name: 'Dummy Restaurant',
+    address: '123 Fake St.',
+    cuisine: 'Fakish',
+    rating: 4.5,
+    phone: '2130001111',
+    image_url: 'http://www.dummyrestaurant.com/pic',
+    website_url: 'http://www.dummyrestaurant.com',
+  },
+  {
+    name: 'Dummy Restaurant 2',
+    address: '345 Fake St.',
+    cuisine: 'Fakish',
+    rating: 4.9,
+    phone: '2130002222',
+    image_url: 'http://www.dummyrestauranttwo.com/pic',
+    website_url: 'http://www.dummyrestauranttwo.com',
+  },
+]
+
 describe('CardScreen', () => {
   afterEach(() => {
     jest.resetAllMocks()
   })
 
   test('displays the first restuarant from fetched restaurants', async () => {
-    const restaurants = [
-      {
-        name: 'Dummy Restaurant',
-        address: '123 Fake St.',
-        cuisine: 'Fakish',
-        rating: 4.5,
-        phone: '2130001111',
-        image_url: 'http://www.dummyrestaurant.com/pic',
-        website_url: 'http://www.dummyrestaurant.com',
-      },
-      {
-        name: 'Dummy Restaurant 2',
-        address: '345 Fake St.',
-        cuisine: 'Fakish',
-        rating: 4.9,
-        phone: '2130002222',
-        image_url: 'http://www.dummyrestauranttwo.com/pic',
-        website_url: 'http://www.dummyrestauranttwo.com',
-      },
-    ]
-
+    const likedRestaurant = jest.fn()
     window.fetch = jest.fn().mockResolvedValue({
       json: () => Promise.resolve({ data: restaurants }),
     })
 
     await act(() => {
-      render(<CardScreen />)
+      render(<CardScreen addLikedRestaurant={likedRestaurant} />)
```

Comment:
nitpick (non-blocking): for the next two tests, I personally wouldn't bother to even create a variable for this mocked prop.  I would just do `<CardScreen addLikedRestaurant={jest.fn()} />` and inline the mock. This communicates that it doesn't matter what the prop is as it doesn't affect the test.